### PR TITLE
limited decimal precision to 2 places on verify.

### PR DESCRIPTION
**Pull request by robline**
addresses TM-848

### CODE REVIEW

**Comment by daaray** [comment]
:palm_tree:

**Comment by robline** [comment]
This one needs to be reviewed again because Colin found more 5-digit instances

File: timepiece/utils/views.py [hunk 1]
```diff
@@ -10,6 +10,12 @@ def class_decorator(View):
     return class_decorator
 
 
+def format_totals(entry_dict, key="sum"):
```
Comment (daaray) [inline timepiece/utils/views.py]: We should have a test that explicitly tests this utility in `timepiece.tests.test_utils`

**Comment by robline** [comment]
ok, this one is updated with the requested test and the latest finds from Colin.

**Pull request merged**